test/integration/node-specific/bson-options/utf8_validation.test.ts

```diff
@@ -16,10 +14,18 @@ const EXPECTED_VALIDATION_ENABLED_ARGUMENT = {
 };
 
 describe('class BinMsg', () => {
+  let deserializeSpy: sinon.SinonSpy;
+
   beforeEach(() => {
+    deserializeSpy = sinon.spy(BSON, 'deserialize');
     deserializeSpy.resetHistory();
```

Comment:
nit: if we're creating a new spy every time, do we need to reset it on creation?

Reply:
fixed!